### ISSUE TITLE
Bookmark folders

### ISSUE THREAD

**Issue by makew0rld** [makew0rld/amfora#56]

**Comment by makeworld-the-better-one** [comment]
#68 must be completed first.

**Comment by SIX10** [comment]
Any reason this was removed from the 1.11.0 milestone?

**Comment by makeworld-the-better-one** [comment]
Nope, I don't remember the reason. Maybe I just judged it wasn't a priority at the time. It's still not a priority for me, but you should know the milestones are loose ideas and not deadlines or carved in stone.

I would happily accept a PR that supports this feature, as it's definitely nice to have.

**Comment by makew0rld** [comment]
I am closing all stale non-bug issues so this issue will be closed. However a PR for this could still be accepted.
